internal/transform/schema.go

```diff
@@ -0,0 +1,38 @@
+package transform
+
+import "time"
+
+//LedgerOutput is a representation of a ledger that aligns with the BigQuery table history_ledger
+type LedgerOutput struct {
+	//sequence number of the ledger
```

Comment:
Put the comment to the right of the field you're referring to, as it reduces ambiguity.